tests/phpunit/tests/meta.php

```diff
@@ -419,6 +419,13 @@ public function test_get_metadata_with_empty_key_nested_array_value() {
 		$this->assertSame( array( $value ), $found['foo'] );
 	}
 
+	public function test_get_metadata_non_existent_object_id() {
+		$this->assertSame( array(), get_metadata( 'user', PHP_INT_MAX ) );
+		$this->assertSame( array(), get_metadata( 'user', PHP_INT_MAX, 'meta_key' ) );
+		$this->assertSame( array(), get_metadata( 'user', PHP_INT_MAX, 'meta_key', false ) );
+		$this->assertSame( '', get_metadata( 'user', PHP_INT_MAX, 'meta_key', true ) );
+	}
```

Comment:
Suggestion: use a data provider and only have a single assertion in the test. This makes for more accurate test counts and easier debugging of tests (if one would fail).

Alternatively, if there is a pertinent reason to have multiple assertions in one test, use the `$message` parameter in each assertion to describe the test failure.

For the data provider, as there are a varying number of arguments, you could use `args` (array) and `expected` as keys and then within the test unpack `$args` with the spread operator, i.e. `get_metadata(...$args)`.

Reply:
Thanks for that update @rodrigoprimo! Next step: add names to the data sets.

Named data sets have three advantages:
1. The name documents what each data set is testing.
2. It makes debugging easier as - in case of a test failure - you won't be confronted with a `with data set #3` (which is fine with a small data provider like this one, but not so much when the data provider has 80+cases and you need to figure out which one is data set `#34`).
3. When using `--testdox`, it will make the test run output much more descriptive.

To change these to named data sets, just add a string key to each data set array. The string key will be the name.

Reply:
Thanks again for your review, @jrfnl. I pushed another commit adding names to the data sets.